src/Entity/ReactionRuleConfig.php

```diff
@@ -208,10 +212,10 @@ public function getTag() {
   }
 
   /**
-   * Returns the event on which this rule will trigger.
+   * Returns the array of events on which this rule will trigger.
```

Comment:
Misses detailed @return documentation. Also, let's better describe the method as "Gets configuration of all events the rule is reacting on." - so it's more clear, that the structure is some configuration structure.

Also, for a lot of code only the event names are from interest (e.g. ReactionRuleStorage), so I think there should be a getter getEventNames() which returns the array of fully qualified event names of the rule.